Sources/MapboxNavigation/InstructionPresenter.swift

```diff
@@ -265,7 +263,7 @@ class InstructionPresenter {
             genericRouteShield.cornerRadius = appearance.cornerRadius
             
             guard let image = takeSnapshot(on: genericRouteShield) else { return nil }
-            spriteRepository.legacyCache.store(image, forKey: key, toDisk: false, completion: nil)
+            spriteRepository.derivedCache.store(image, forKey: key, toDisk: true, completion: nil)
```

Comment:
Because the `legacyCache` we used to have is used to store all the legacy shield icons and the generated exit views and generic shields, along with the SpritInfoCache and the SpriteCache. Right now we have the `derivedCache` to store the generated shields and the single cut Sprite shield icons. So they have different functions. And also now it defaults to save the generated ones to disk in the `ImageCache`, which would avoid the re-downloading when the memory capacity is reached and memory is cleared out.